src/Heading.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function Heading() {
+  return (
+    <header className="App-header">
+      {" "}
+      CYF Hotel <Logo />
+    </header>
+  );
+}
+
+function Logo() {
```

Comment:
Normally we like to have every separate component in it's own file. Where do you think we should move this code to?